test/e2e/operator_test.go

```diff
@@ -421,6 +420,30 @@ func componentRefEventuallyExists(w watch.Interface, exists bool, ref *corev1.Ob
 	}))
 }
 
+func multipleComponentRefsEventuallyExists(w watch.Interface, exists bool, refs []*corev1.ObjectReference) {
+	deadline, cancel := context.WithTimeout(context.Background(), 1*time.Minute)
+	defer cancel()
```

Comment:
Hmm let the test case handle this instead, and take a context.Context parameter? Thoughts?

Reply:
I don't have a strong opinion about this.  It's the same way as the `componentRefEventuallyExists` function.  I can refactor it, too.